### ISSUE TITLE
New Install Doesn't Work with 4.1.0

### ISSUE THREAD

**Issue by MosheTzvi** [rospogrigio/localtuya#1024]
## The problem

First time adding localtuya to a HASS install via HACS.  Files were copied correctly, but no config ever launches.  Instructions "To start configuring the integration, just press the "+ADD INTEGRATION" button in the Settings - Integrations page, and select LocalTuya from the drop-down menu."  Aren't relevant as HACS has no "+ADD INTEGRATION" button.

## Environment

- Localtuya version:  4.1.0
- Last working localtuya version (if known and relevant): none
- Home Assistant Core version:  latest
- [] Are you using the Home Assistant Tuya Cloud component ? yes
- [] Are you using the Tuya App in parallel ? yes

## Steps to reproduce

1.install localtuya from HACS
2. 
3.

## Configuration `configuration.yaml` or `config_flow`

```yaml

```

## Provide Home Assistant taceback/logs

```txt

```

**Comment by bmarr** [comment]
'Aren't relevant as HACS has no "+ADD INTEGRATION" button.'

The instructions refer not to HACS, but to the HA sidebar, Settings -> Devices and Services
The +ADD INTEGRATION button is in the lower right corner of that page.

However, I am also installing localTuya for the first time and after clicking on the +ADD INTEGRATION button, does not show up in the "Set up a new integration" list, localTuya is not found on the list of available integrations.

My setup is hassio running in a KVM on TrueNAS-22.02.2.1:

Home Assistant 2022.9.0
Supervisor 2022.08.6
Operating System 8.5
Frontend 20220907.0 - latest

HACS:
Integration version:	1.27.1
Frontend version:	20220825160758

localTuya 4.1.0

**Comment by MosheTzvi** [comment]
As you wish. LocalTuya is not showing up on the available integrations list.


On Sep 8, 2022, at 22:11, Braden marr ***@***.***> wrote:

﻿

'Aren't relevant as HACS has no "+ADD INTEGRATION" button.'

The instructions refer not to HACS, but to the HA sidebar, Settings -> Devices and Services
The +ADD INTEGRATION button is in the lower right corner of that page.

However, I am also installing localTuya for the first time and after clicking on the +ADD INTEGRATION button, does not show up in the "Set up a new integration" list, localTuya is not found on the list of available integrations.

My setup is hassio running in a KVM on TrueNAS-22.02.2.1:

Home Assistant 2022.9.0
Supervisor 2022.08.6
Operating System 8.5
Frontend 20220907.0 - latest

HACS:
Integration version: 1.27.1
Frontend version: 20220825160758

localTuya 4.1.0

**Comment by bmarr** [comment]
> Make sure you reboot HA after install the integration. Sometimes it is also necessary to refresh browser cache (Shift+F5 / Ctrl+F5)

Yes, refresh browser cache AND log out your HA user and log back in. The combination of doing these two things worked for me.
